src/skyparams.h

```diff
@@ -118,4 +141,16 @@ class SkyboxDefaults
 		stars.scale = 1;
 		return stars;
 	}
+
+	const CloudParams getCloudDefaults()
```

Comment:
Idea: Make these functions static. They do not need access to an initialized `SkyboxDefaults` instance.